README.txt

```diff
@@ -1 +1,24 @@
-## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
\ No newline at end of file
+## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
+
+**How to Earn XP**
+
+**Earn Activity XP**
+
+- As a lender, provide your first loan (+1XP for every 5th)
+- As a borrower, pay interest on your first loan (+1XP for every 5th)
+- Approve your first lending strategy (+1XP for every 5th)
+- Lend or borrow the highest loan amount in a given month
+
+**Earn Contributor XP**
+
+- Leave your first constructive comment on Github (+1XP for every 5th)
```

Comment:
same as below re "meaningful", here issue being 'constructive'. But this one is harder to be dumb about so maybe we always need to have some human check here. Could also be like per "PR reviewed" rather than comment